src/main/java/com/emc/mongoose/storage/driver/kafka/KafkaStorageDriver.java

```diff
@@ -401,7 +399,48 @@ void consumeRecords(final List<O> recOps) {
     }
   }
 
-  void createTopics(final List<O> topicOps) {}
+  void createTopics(final List<O> topicOps) {
+    val nodeAddr = topicOps.get(0).nodeAddr();
+    try {
+      val config = configCache.computeIfAbsent(nodeAddr, this::createAdminClientConfig);
+      val adminClientCreateFunc =
+          adminClientCreateFuncCache.computeIfAbsent(config, AdminClientCreateFunctionImpl::new);
+      val adminClient = adminClientCache.computeIfAbsent(nodeAddr, adminClientCreateFunc);
+      for (var i = 0; i < topicOps.size(); i++) {
```

Comment:
why not to invoke createTopics for all ?

Reply:
Because we need to process items' statuses correctly. Successful status must be given only to items which topics were created.